src/GraphBLAS-sharp/Objects/Vector.fs

```diff
@@ -0,0 +1,37 @@
+namespace GraphBLAS.FSharp
+
+type VectorType =
+    | COO
+    | Bitmap
+
+type Vector<'a when 'a : struct> =
+    | VectorCOO of COOVector<'a>
+    | VectorBitmap of BitmapVector<'a>
+
+and COOVector<'a> =
+    {
+        Size: int
+        Indices: int[]
+        Values: 'a[]
+    }
+
+    override this.ToString() =
+        [
+            sprintf "Sparse Vector\n"
+            sprintf "Size:    %i \n" this.Size
+            sprintf "Indices: %A \n" this.Indices
+            sprintf "Values:  %A \n" this.Values
+        ]
+        |> String.concat ""
+
+and BitmapVector<'a> =
+    {
+        Bitmap: bool[]
```

Comment:
Should we use [BitArray](https://docs.microsoft.com/ru-ru/dotnet/api/system.collections.bitarray?view=net-5.0) instead?